### ISSUE TITLE
How to at-mention legit users in an Adaptive Card, better without the Flow[Bot]?

### ISSUE THREAD

**Issue by IdiotLeon** [MicrosoftDocs/msteams-docs#1058]
The experience I had is to display "&lt;at&gt;blahblah@example.com&lt;/at&gt;" as literal texts.

(The email address actually sent is valid. The above dummy email address is just for safety's concern.)

Here is the JSON sent as a Channel message

```
{
    "contentType": "application/vnd.microsoft.card.adaptive",
    "content": {
        "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
        "type": "AdaptiveCard",
        "version": "1.0",
        "body": [
            {
                "type": "Container",
                "items": [
                    {
                        "weight": "bolder",
                        "size": "medium",
                        "text": "<at>blahblah@example.com</at> Hello World",
                        "wrap": false,
                        "type": "TextBlock"
                    }
                ]
            }
        ],
        "actions": []
    }
}
```

Better to use my own bot, instead of the Flow[bot].

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a8a3bcf2-332a-8d24-a8d1-7e3816897745
* Version Independent ID: 68fb9551-ce95-0112-2703-721a1af01f23
* Content: [Cards Reference - Teams](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/cards/cards-reference)
* Content Source: [msteams-platform/concepts/Cards/cards-reference.md](https://github.com/MicrosoftDocs/msteams-docs/blob/master/msteams-platform/concepts/Cards/cards-reference.md)
* Product: **msteams**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by Gousia-MSFT** [comment]
@IdiotLeon Currently, it is not possible to at-mention a user in an Adaptive card. You could mention in a  [text message](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/bots/bot-conversations/bots-conv-channel#constructing-mentions) but not in card. This is something we are working on but we do not have an ETA to share with you.

**Comment by IdiotLeon** [comment]
@Gousia-MSFT  thanks for your quick response and confirmation.
However, I also find it difficult to at-mention users in regular text messages. The example provided in the doc seems to work only on .NET Framework, instead of .NET Core. To be exact: `addMentionToText() `seems to exist in `Microsoft.Bot.Connector.Teams 0.9.0`, yet to be included in prerelease version `4.3.0-beta1` for .NET Core 2.1. Please take a look at another question from me [here](https://github.com/MicrosoftDocs/msteams-docs/issues/1057).

Basically I just want to at-mention people from my notification bot.

**Comment by Gousia-MSFT** [comment]
@IdiotLeon Please see the comment from clearab on the other issue.
